### PR TITLE
[REL-516304] - Relativity.Processing.SDK

### PR DESCRIPTION
**Pull request by gelatingigalo**
Updating Processing version information and release notes.

### CODE REVIEW

**Review by jake-martin — APPROVED** [review]

File: Relativity.Processing.SDK.md [hunk 1]
```diff
@@ -4,20 +4,41 @@
 
 This package contains interfaces for the public APIs of the Relativity Processing application.
 
-## 11.3.101.10
+## 12.2.13
 
 ### Release Notes
 
-* Initial version
```
Comment (Bbuff101) [inline Relativity.Processing.SDK.md:11]: This seems like a lot of information for interface changes. Should we only be including references specific to our service interfaces?

File: Relativity.Processing.SDK.md [hunk 2]
```diff
@@ -4,20 +4,41 @@
...
 
-* Initial version
+* Updated DFV to properly populate Processing Deletion fields.
+* Ported Post Publish Delete performance improvement changes from mayapple.
+* Bumped the Invariant Services package for ARM statistics endpoint defect fix. Fixed error field removals.
+* Fix adding index to zprocessingdochistory table.
+* Advanced Picker pagination support in files tab.
+* Removing toggle for Processing Profile Inclusion/Exclusion feature.
+* ARM Processing: Do not throw exception for duplicate cancel requests for canceled or canceling workspaces and Fix Processing Exists readiness check defect.
+* Replaced remaining WCF endpoints with Kepler endpoints.
```
Comment (Bbuff101) [inline Relativity.Processing.SDK.md:18]: This should be removed as it's specific to Invariant infrastructure and isn't something the RAP exposes.

File: Relativity.Processing.SDK.md (hunk 1 above)
Comment (Bbuff101) [inline Relativity.Processing.SDK.md:11]: We **should** call out that we removed the Processing Profile Manager API endpoint and that consumers should now use Object Manager API.

Comment (Bbuff101) [inline Relativity.Processing.SDK.md:11]: We should probably also include a note regarding the additional of a versioned **v1** API surface and that consumers are highly encouraged to transition to those APIs.

Comment (Bbuff101) [inline Relativity.Processing.SDK.md:11]: We **should** call out that we removed the Processing Profile Manager API endpoint and that consumers should now use Object Manager API.

**Review by Bbuff101 — CHANGES_REQUESTED** [review]

**Review by Bbuff101 — CHANGES_REQUESTED** [review]

File: Relativity.Processing.SDK.md [hunk 3]
```diff
@@ -4,20 +4,41 @@
...
 
 ### Release Notes
 
-* Initial version
+* Updated DFV to properly populate Processing Deletion fields.
+* Ported Post Publish Delete performance improvement changes from mayapple.
+* Bumped the Invariant Services package for ARM statistics endpoint defect fix. Fixed error field removals.
+* Fix adding index to zprocessingdochistory table.
+* Advanced Picker pagination support in files tab.
+* Removing toggle for Processing Profile Inclusion/Exclusion feature.
```
Comment (uladzimir-yushkevich) [inline Relativity.Processing.SDK.md]: This is cleanup work, and have no functional impact for consumers. I would suggest removing from release notes.

File: Relativity.Processing.SDK.md [hunk 4]
```diff
@@ -4,20 +4,28 @@
...
 
-* Initial version
+* Bumped the Invariant Services package for ARM statistics endpoint defect fix. Fixed error field removals.
+* Added a NuGet package for Password Bank, renamed Password Bank tab, and moved from ApplicationBase.
+* Resolved dependency injection issues and removed low-use RSAPI code, along with legacy test projects.
+* Removed redundant Kepler endpoint check for ValidateApiEndpoints API, removed ProcessingBase package and consolidated projects into Relativity.Eca.
+* Added a new private endpoint for updating count after Post Publish Delete job.
+* Use Object Manager instead of RSAPI for Custodian, Processing Set and Processing Data Source updates.
+* Started using new Shared.ApplicationBase package with contract changes to IContextContainer.
+* Removed the Processing Profile Manager API endpoint, consumers should now use Object Manager API.
```
Comment (dakkerns) [inline Relativity.Processing.SDK.md:18]: Could also mention that all calls to Invariant WCF endpoints have been replaced with Kepler equivalents. No WCF is used by the Processing APIs whatsoever now.

File: Relativity.Processing.SDK.md [hunk 5]
```diff
@@ -4,20 +4,41 @@
...
+* Added a NuGet package for Password Bank, renamed Password Bank tab, and moved from ApplicationBase.
+* Added Dedupe Status Field to Files tab default view.
+* Resolved dependency injection issues and removed low-use RSAPI code, along with legacy test projects.
+* Removed redundant Kepler endpoint check for ValidateApiEndpoints API, removed ProcessingBase package and consolidated projects into Relativity.Eca.
+* Added a new private endpoint for updating count after Post Publish Delete job.
+* Added APM to PreMassDeleteEH.
+* Enabled OI PowerPoint toggle.
+* Moving Processing Set page interactions to web assets from MVC.
+* Use Object Manager instead of RSAPI for Custodian, Processing Set and Processing Data Source updates.
+* Updates for Error Workflow, unhide Document Errors tab.
```
Comment (uladzimir-yushkevich) [inline Relativity.Processing.SDK.md]: Maybe rephrase this, I don't think "unhide Document Errors tab" applies to 11.3.101.10 -> 12.2.13

File: Relativity.Processing.SDK.md (hunk 3 above)
Comment (gelatingigalo) [reply Relativity.Processing.SDK.md]: Should be removed

**Review by Bbuff101 — APPROVED** [review]

**Review by jake-martin — APPROVED** [review]
